src/components/theme/Sitemap/Sitemap.jsx

```diff
@@ -106,25 +102,20 @@ export default compose(
   connect(
     (state) => ({
       items: state.navigation.items,
-      lang: state.intl.locale,
+      navroot: state.navroot.data,
     }),
-    { getNavigation },
+    { getNavigation, getNavroot },
   ),
   asyncConnect([
     {
       key: 'navigation',
       promise: ({ location, store: { dispatch, getState } }) => {
         if (!__SERVER__) return;
-        const { settings } = config;
 
-        const path = getSitemapPath(
-          location.pathname,
-          settings.isMultilingual
-            ? toBackendLang(getState().intl.locale)
-            : null,
+        const navroot = getState().navroot.data?.navroot?.navroot?.['@id'];
```

Comment:
Is the navroot available in both 7.x and 8.x branches of plone.restapi?

Reply:
No right now it is not. But I can try to add a PR to backport them .

Reply:
Why does it need to be backported? This fix will go in Volto 17 and maybe Volto 16, which both expect plone.restapi 8.x, even on Plone 5.

Reply:
https://github.com/plone/plone.restapi/blob/ed192208499989b379de8534a3aa12d037316db4/setup.py#L11

Volto 16 works fine with Plone 4, we're using it in production on a complex site. If this PR is backported to Volto 16, it will be a breaking change.

Reply:
Okay, I didn't know you are using Volto 16 with Plone 4. That is not officially supported, as far as I am aware, and there are no automated tests for it in volto. So this is a risk you took on in using Volto 16 with an unsupported Plone version. But, I agree we should avoid breaking that, unless it's really hard to avoid.

Reply:
The navroot endpoints are not that complex, I would say that they could be backported to 7.x and 8.x easily. But let's see what the tests say :sweat_smile:

Reply:
See: https://github.com/plone/plone.restapi/pull/1702